Dockerfile

```diff
@@ -23,9 +23,9 @@ COPY . /opt/vmware_exporter/
 
 RUN set -x; buildDeps="gcc python-dev musl-dev libffi-dev openssl openssl-dev" \
  && apk add --no-cache --update $buildDeps \
- && pip install -r requirements.txt \
+ && pip install -r requirements.txt . \
  && apk del $buildDeps
 
 EXPOSE 9272
 
-ENTRYPOINT ["python", "-u", "/opt/vmware_exporter/vmware_exporter/vmware_exporter.py"]
+ENTRYPOINT ["/usr/local/bin/vmware_exporter"]
```

Comment:
I think this is great, my only comment is that in the previous "script-like" execution we passed on the `-u` flag to python which forced python to send stream to stdout unbuffered, this is mainly to allow log entries to show immediately and not when buffer is flushed.

I think we can achieve the same by setting `PYTHONUNBUFFERED=1` ENV variable, wdyt?

Reply:
Yes, I set that environment var on most of my own containers, we should deffo do that.